kolibri/auth/migrations/0002_auto_20160203_1516.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yeah, it was weird... I had to disable it, then autoreload, then enable it again. Not sure what's up. But it works here.